config/express.js

```diff
@@ -24,15 +24,15 @@ module.exports = function(app, passport, mongoose) {
 
     //Don't use logger for test env
     if (process.env.NODE_ENV !== 'test') {
-        app.use(express.logger('dev'));
+        app.use(express.logger('dev');
```

Comment:
You need to pay attention to this. It might cause problems in the entire application. Its seems one closing brace is missing